test/e2e/containers/prow-npd-e2e/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/k8s-testimages/kubekins-e2e:v20190607-c90971a-master
```

Comment:
How do we keep this up to date when `kubekins-e2e` is updated?

Reply:
Good question. Actually, I'm not even 100% sure that we need this container at all.

All it does is this:
```
  git clone https://github.com/kubernetes/node-problem-detector.git

  pushd node-problem-detector

  ZONE=us-central1-a PROJECT=xueweiz-experimental \
  VM_IMAGE=cos-73-11647-217-0 IMAGE_PROJECT=cos-cloud \
  SSH_USER=${USER} SSH_KEY=${JENKINS_GCE_SSH_PRIVATE_KEY_FILE} \
  make e2e-test

  popd
  rm -rf node-problem-detector
```

The reason I used it, is because it made my initial Prow setup much easier and faster. Now that it's more-or-less stabilized, I think we can get rid of it now, as I believe all the above steps can be done in the Prow job config. I will confirm with Sen Lu later on.

Let me exclude it in this PR for now. If we really still need it, I will submit a PR for it later.